package.json

```diff
@@ -51,6 +50,12 @@
   },
   "dependencies": {
     "aws-sdk": "~2.2.28",
+    "babel-cli": "^6.24.1",
```

Comment:
Can the `babel-*` packages be development dependencies, or are they needed at run time? If they're only needed to run the transpile process (and aren't needed at run time) can we do something like
```bash
npm install
npm run transpile
npm prune --production
``` 
before copying the ./node_modules folder for packaging?